orchagent/dash/dashaclorch.cpp

```diff
@@ -285,16 +285,16 @@ sai_attr_id_t getSaiStage(DashAclDirection d, sai_ip_addr_family_t f, DashAclSta
     return stage->second;
 }
 
-DashAclOrch::DashAclOrch(DBConnector *db, const vector<string> &tables, DashOrch *dash_orch) :
-    Orch(db, tables),
+DashAclOrch::DashAclOrch(DBConnector *db, const vector<string> &tables, DashOrch *dash_orch, ZmqServer *zmqServer) :
+    Orch(db, tables, zmqServer),
```

Comment:
According to the function addConsumer in orch.cpp, original redis db and new zmqServer are mutually exclusive. If yes, why did this contractor receive both DBConnector and ZmqConnector?

Reply:
This is because ZmqConsumerStateTable need use db connector to write received data to redis:
So the ctor of ZmqConsumerStateTable need a DBConnector as parameter:

    ZmqConsumerStateTable(DBConnector *db, const std::string &tableName, ZmqServer &zmqServer, int popBatchSize = DEFAULT_POP_BATCH_SIZE, int pri = 0, bool dbPersistence = true);

Reply:
Fixed, add ZmqOrch class. all dash orch will inherit from ZmqOrch.